.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -47,6 +47,7 @@ steps:
       bash .ci/scripts/setup_sklearn.sh $(SKLEARN_VERSION)
       pip install --upgrade -r requirements-test.txt -r requirements-test-optional.txt
       pip install $(python .ci/scripts/get_compatible_scipy_version.py)
+      if [ $(echo $(PYTHON_VERSION) | grep '3.8\|3.9\|3.10') ]; then conda install -q -y -c intel dpnp; fi

```

Comment:
```suggestion
```
Might make more sense to update line 28:
`else export DPCPP_PACKAGE="dpctl>=0.14 dpcpp-cpp-rt>=2023.1.0" dpnp`

Reply:
No. DPNP is not build dependency. this is runtime, test dep.